manage.py

```diff
@@ -241,6 +241,14 @@ def toggle_site_status():
         print('Done!')
 
 
+@cli.command(name='set_rate_limits')
+@click.argument('per_ip')
+@click.argument('per_token')
+@click.argument('window_size')
+def set_rate_limits(per_ip, per_token, window_size):
```

Comment:
we don't have token auth in AB, so how about we remove this parameter (and optionally set it to the same as per_ip?)
Please add documentation and types to the arguments.
It'd make sense to also print what the parameters are set to when you run this function.
Do we want any sanity checking to make sure that these settings aren't set to anything really strict? e.g. a warning if the effective limit goes lower than 1 per second